web/packages/teleterm/src/services/config/getKeyboardShortcutSchema.ts

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { z } from 'zod';
+
+import { Platform } from 'teleterm/mainProcess/types';
+
+export function invalidKeyCodeIssue(wrongKeyCode: string): z.IssueData {
+  return {
+    code: z.ZodIssueCode.custom,
+    message: `"${wrongKeyCode}" cannot be used as a key code.`,
+  };
+}
+
+export function invalidModifierIssue(wrongModifiers: string[]): z.IssueData {
+  return {
+    code: z.ZodIssueCode.custom,
+    message: `${wrongModifiers
+      .map(m => `"${m}"`)
+      .join(', ')} cannot be used as a modifier.`,
```

Comment:
I feel like if there's at least one of those errors in the config file, we should include the list of valid modifiers for the platform. This message alone is just not that helpful. We can do that in a separate PR though.

I feel like it's not uncommon to expect "Command" to work and then be surprised when it doesn't, with no clear error message.

Reply:
Done.